python: can't open file '/mnt/d/X/Programming/idola/idola_api/idola_bot.py': [Errno 2] No such file or directory
python: can't open file 'idola_bot.py': [Errno 2] No such file or directory
python: can't open file 'idola_bot.py': [Errno 2] No such file or directory
python3: can't open file 'idola_bot.py': [Errno 2] No such file or directory
python3: can't open file 'idola_bot.py': [Errno 2] No such file or directory
